packages/core/logger/src/Logger.js

```diff
@@ -60,6 +86,44 @@ class Logger {
 const logger = new Logger();
 export default logger;
 
+export type PluginLoggerOpts = {|
+  origin: string
+|};
+
+export class PluginLogger {
```

Comment:
if we're not going to expose a default `@parcel/logger` anymore, then maybe we should get rid of that package, move it into core, and put `PluginLogger` in the public folder like the other public objects?

Reply:
Currently we do, but eventually it would be nice to pass a pluginLogger instance to each plugin call. Not entirely sure how to do it yet though. Creating PluginOptions per call seems expensive?